graphql/resolvers.go

```diff
@@ -2502,29 +2504,19 @@ func (r *taskResolver) CanModifyAnnotation(ctx context.Context, obj *restModel.A
 		Permission:    evergreen.PermissionAnnotations,
 		RequiredLevel: evergreen.AnnotationsModify.Value,
 	}
-	return authUser.HasPermission(permissions), nil
-
-}
-
-// to be removed
-func (r *annotationResolver) UserCanModify(ctx context.Context, obj *restModel.APITaskAnnotation) (*bool, error) {
```

Comment:
I just removed this cause I saw the comment I'd left; this was dead code. If this is actually an involved thing to remove though I'll just leave it in and make sure it's ticketed.

Reply:
As long as that field is not used in any graphql queries it should be safe to remove